templates/donor-donation-success.html.twig

```diff
@@ -54,6 +54,12 @@
 
   <p>Total amount you have been charged is <strong>{{ totalChargedAmount | format_currency(currencyCode) }}</strong>.</p>
 
+  {% if createAccountUri is not empty %}
+    <p>For a smoother giving experience, create a Big Give account. Your information will be saved
+    securely, making future donations quicker and easier.</p>
+    <p><strong><a href="{{ createAccountUri }}">Create a Big Give donor account</a></strong>.</p>
+  {% endif %}
```

Comment:
Looks good, but if you just read this and don't examine the URI there's no way to know that it's got a magic token in it - I think we need to tell them that this is a special link that gives access to the account details (especially payment card) that they already entered, and that it will expire in 8 hours.

We can't be sure people will read the email but if they do they should know that this link shouldn't be shared within that time since it gives that access.

(I think I made a mistake not putting the expiry date on the actual token object, but now I'm a bit loathe to go back into Identity Messages and Matchbot to add that)

Reply:
Yeah, I think we do need something, probably without too much detail about what it provides. Had a go in https://github.com/thebiggive/mailer/pull/156/commits/1e3f5a33781212b5593a8e80a272c9c304712056